.github/workflows/main.yml

```diff
@@ -43,3 +43,20 @@ jobs:
         continue-on-error: false
         env:
           LIGHTGBM_EXAMPLES_PATH: LightGBM-2.3.1
+ docs:
```

Comment:
So this one is the reason why the CI isn't working -- the indentation for `docs:` needs to be 2, not 1, then it should trigger the CI correctly.

I think we need to add something like this:

```
  docs:
    on:
      push:
        branches: master
        tags: *
```